### PR TITLE
Update tutorial for tech writer comment

### PR DESCRIPTION
**Pull request by kumaripriyankasingh**
Updating `tutorial.md` for [these review comments](https://github.com/GoogleCloudPlatform/solutionbuilder-terraform-google-three-tier-web-app/pull/3/files#diff-54d121b9f8f94af2cb3fa5d2bbb327559d7145a8e5f45a98cc944cf204247cd8).

### CODE REVIEW

File: tutorial.md [hunk 1]
```diff
@@ -18,32 +18,33 @@ Open the directory where the repository is cloned as a workspace in the editor,
...
 
 Both the frontend service and the middleware service are built as container images and deployed using Cloud Run.
-
+We also strongly recommend that you familiarize yourself with the three-tier web app solution by reading the [solution guide](https://cloud.google.com/architecture/application-development/three-tier-web-app).
 
 ## Edit the solution
 
-Edit the `createHandler` function in <walkthrough-editor-select-line filePath="./src/middleware/main.go" startLine="170" endLine="171" startCharacterOffset="0" endCharacterOffset="0">./src/middleware/main.go</walkthrough-editor-select-line> to add a prefix string to every TODO item by replacing `t.Title = r.FormValue("title")` with `t.Title = "Prefix " + r.FormValue("title")`.
+For example, edit the createHandler function in <walkthrough-editor-select-line filePath="./src/middleware/main.go" startLine="170" endLine="171" startCharacterOffset="0" endCharacterOffset="0">./src/middleware/main.go</walkthrough-editor-select-line> to add a prefix string to every TODO. To add the prefix, replace `t.Title = r.FormValue("title") with t.Title = "Prefix " + r.FormValue("title")`.
```
Comment (ayushmjain) [inline tutorial.md:46]: `t.Title = r.FormValue("title") with t.Title = "Prefix " + r.FormValue("title")` the word "with" shouldn't be inside the backticks

File: tutorial.md [hunk 2]
```diff
@@ -18,32 +18,33 @@ Open the directory where the repository is cloned as a workspace in the editor,
...
```
Comment (ayushmjain) [inline tutorial.md:45]: why are we removing backticks from createHandler?

Comment (kumaripriyankasingh) [reply tutorial.md:45]: Please check the file diff again. We are not removing backtick.

Comment (ayushmjain) [reply tutorial.md:45]: Screenshot 2023-12-13 at 14 10 26
the backtick has been removed from the word createHandler word in the first line

File: tutorial.md (hunk 1 above)
Comment (kumaripriyankasingh) [reply tutorial.md:46]: Fixed now.

**Review by ayushmjain — APPROVED** [review]

**Pull request merged**